From: Boyuan Yang <073plan@gmail.com>
Date: Sat, 1 Oct 2016 15:35:31 +0800
Subject: Makefile: do not handle binary installation

The upstream installed the binary into /usr/share, which is
unacceptable. Trying to patch and use dh_install to handle it.
---
 Makefile                   | 2 +-
 install_script/copy_bin.sh | 3 ---
 2 files changed, 1 insertion(+), 4 deletions(-)
 delete mode 100755 install_script/copy_bin.sh

diff --git a/Makefile b/Makefile
index f4c4855..d1200b5 100644
--- a/Makefile
+++ b/Makefile
@@ -108,7 +108,7 @@ main: ./lib/animation.vala \
 install:
 	./install_script/copy_resource.sh
 	./install_script/copy_icons.sh
-	./install_script/copy_bin.sh
+	#./install_script/copy_bin.sh
 	./install_script/update_mo.sh
 
 clean:
diff --git a/install_script/copy_bin.sh b/install_script/copy_bin.sh
deleted file mode 100755
index e6102b5..0000000
--- a/install_script/copy_bin.sh
+++ /dev/null
@@ -1,3 +0,0 @@
-#!/bin/sh
-install -p -D -m 0755 main ${DESTDIR}${PREFIX}/share/deepin-terminal
-ln -sf ${PREFIX}/share/deepin-terminal/main ${DESTDIR}${PREFIX}/bin/deepin-terminal
